staging/src/github.com/kubeedge/mapper-framework/pkg/grpcserver/device.go

```diff
@@ -51,7 +51,7 @@ func (s *Server) RegisterDevice(ctx context.Context, request *dmiapi.RegisterDev
 	}
 
 	deviceInstance.PProtocol = protocol
-	s.devPanel.UpdateDev(&model, deviceInstance, &protocol)
+	s.devPanel.UpdateDev(&model, deviceInstance)
```

Comment:
I have made these changes in https://github.com/kubeedge/kubeedge/pull/5292, do you need to rebase?

Reply:
I have rebased them @cl2017